ParisBNF/abb/BNFabb160.xml

```diff
@@ -0,0 +1,329 @@
+                                     አወቅኽ፡ አእመረ፡ አወቀ፡ አእመርክሙ፡ ብ፡ አወቃችሁ፡
+                                     </incipit>
+                               </msItem>
+
+                               <msItem xml:id="ms_i5">
+                                   <locus from="65" to="73"/>
+                                   <title type="complete" ref="LIT6511Sawasew" cert="low">Lexicon
+                                     of verbs and poetic formulas (<foreign xml:lang="gez">ሰዋስወ፡ ግስ፡</foreign>)</title>
+                                   <textLang mainLang="gez"/>
+                                   <incipit xml:lang="gez">
+                                   <hi rend="rubric">እጽሕፍ፡ ሰዋስወ፡ <add place="above">ግስ፡</add></hi> ፍኖተ፡ ጼሐ፡ ፍኖተ፡ ተመርሐ፡ በግዓ፡
+                                   ዘብሐ፡ ላህመ፡ ጠብሐ፡ ቀሰመ፡ ልሱሐ፡ ሐነጸ፡ ጽርሐ፡ ቀደሰ፡ ርሱሐ፡ ሞጣሕተ፡ ተሞጥሐ፡ ሀብለ፡ አንኃ፡ ቀርነ፡ ነፍሐ፡ ወርቀ፡ አርብሐ፡ ማኅተመ፡ ፈትሐ፡ ዕደዊሁ፡ ሙቁሐነ፡
+                                   ፈትሐ፡ ዓለም፡ ረብሐ፡
+                                 </incipit>
+                                   <note>In terms of structure, similar to <ref type="work" corresp="LIT6511Sawasew"/>,
+                                 but only a part of the verbs and expressions is the same.</note>
+                               </msItem>
+
+                             </msItem>
+
+                          </msContents>
+
+                          <physDesc>
+                             <objectDesc form="Codex">
+                                <supportDesc>
+                                   <support>
+                                      <material key="parchment"/>
+                                   </support>
+                                   <extent>
+                                      <measure unit="page">86 (i-vii + 73 + 74-79)</measure>
+                                      <measure unit="page" type="blank">12</measure>
+                                        <locus target="#i #ii #iii #v #vi #vii #74 #75 #76 #77 #78 #79"></locus>
+                                        <note>The catalogues provide somewhat varying
+                                          information on the number of leaves/pages.</note>
+                                      <dimensions type="outer" unit="mm">
+                                         <height>95</height>
+                                         <width>80</width>
+                                      </dimensions>
+                                   </extent>
+
+                                   <foliation>The manuscript is paginated, the page numbers
+                                     are written in the outer corner of the upper margin.
+                                     Leaves at the beginning and the end of the volume with no page numbers have been paginated as
+                                    <locus target="#i #ii #iii #v #vi #vii"/>
+                                     and <locus target="#74 #75 #76 #77 #78 #79"/>.
```

Comment:
locus only takes folio numbers, not page numbers, make sure you recalculate

Reply:
ok